docs/migrating-a-legacy-transformer.md

```diff
@@ -92,12 +89,39 @@ An example of this can be seen in the **HelloWorldController.java**.
 This is equivalent to the **ContentReader** parameter.
 * The response from the `/transform` endpoint contains the transformed file.
 This is equivalent to the **ContentWriter** parameter.
-* Requests to a T-Engine's `/transform` endpoint contain a list of
-transform options as defined by the [T-Engine configuration](creating-a-t-engine.md#t-engine-configuration).
-These are equivalent to the options in the **TransformationOptions** parameter.
 
+Depending on the case, on a legacy Transformer, new transform options need to
+subclass `TransformationOptions`, or implement `TransformationSourceOptions`
+which needs to be marshaled and un-marshaled. After custom
+options are defined, `repo.renditions2.TransformationOptionsConverter` needs to be subclassed,
+in which the logic to handle the new options is defined and injected into Alfresco-repository
+via a Spring Beans.
 
+* Requests to a T-Engine's `/transform` endpoint contain a list of
+transform options as defined by the [T-Engine configuration](creating-a-t-engine.md#t-engine-configuration).
+These are equivalent to the options in the **TransformationOptions** parameter
+see [engine_config](https://github.com/Alfresco/alfresco-transform-core/blob/master/docs/engine_config.md)
+for more information.
+
+**Pipeline Transformers:**
+
+Pipeline Transformers definitions for the Legacy Transformers was done by adding properties in
+alfresco-global.properties. Definition for the pipline is `Transformer1 | Extension | Transformer2`,
+the resulting pipeline transformer will have the same supportedExtension as Transformer1, but the resulted
+targetExtension will be the sum of targetExtension(Transformer1) + targetExtension(Transformer2(Extension)).
+Additional properties are available:
+* `.extension.Ext1.Ext2.supported=false` restricts the transformation from Ext1 to Ext2 for a specific Transformer.
+* `.priority=200` sets priority value of the transformer, the values are like the order in a queue,
+the lower the number the higher the priority is.
+* `.available=false` disables a transformer.
+```
+# alfresco-pdf-renderer.ImageMagick
+# ---------------------------------
+# content.transformer.alfresco-pdf-renderer.ImageMagick.pipeline=alfresco-pdf-renderer|png|ImageMagick
+# content.transformer.alfresco-pdf-renderer.ImageMagick.priority=200
+# content.transformer.alfresco-pdf-renderer.ImageMagick.available=false
+# content.transformer.alfresco-pdf-renderer.ImageMagick.extension.ai.jpg.supported=false
 
-TODO
-* Talk about not needing to create a sub class of TransformOptions, or
-  the need to marshal and un marshal TransformOptions.
+```
+Pipeline definitions for Local Transformers are done via JSON rather than alfresco-global.properties
+see section [Configure a custom transform pipeline](https://github.com/Alfresco/acs-packaging/blob/master/docs/custom-transforms-and-renditions.md#configure-a-custom-transform-pipeline).
```

Comment:
We should add a priority to the example json pipeline..

Reply:
Also, should explain how the other options that you listed equate to the json format, for example the supported extension, in the json format you just list the ones you want to support. This means you don't have to explicitly say what you don't support.